docs/contributors/javascript-testing.md

```diff
@@ -4,9 +4,18 @@ Tests for JavaScript in the Blocks plugin are powered by [Jest](https://jestjs.i
 
 The Blocks plugin follows the same patterns as Gutenberg, therefore for instructions on writing tests you can [refer to this page in the Gutenberg Handbook](https://developer.wordpress.org/block-editor/contributors/develop/testing-overview/).
 
-## Running Tests
+We have two kinds of JavaScript tests:
 
-Assuming you've already followed the [Getting Started Guide](getting-started.md) on setting up node and other dependencies, tests are ran from the command line using the following command:
+- JavaScript unit tests - test APIs, hooks, library functionality that we use to build blocks or expose to plugin authors.
+- End-to-end (e2e) tests - test blocks from the user interface. 
+
+These tests are all run automatically on open PRs by Travis CI.
+
+## How to run JavaScript unit tests
+
+Unit tests are implemented near the code they test, in `*.test.js` files.
+
+Assuming you've already followed the [Getting Started Guide](getting-started.md) on setting up node and other dependencies, unit tests are ran from the command line using the following command:
```

Comment:
I noticed `composer install` is required for e2e tests too, so I think it would be good to move this line outside the unit tests section. Probably above so it's clear dependencies are needed for both, unit and e2e tests?

Reply:
Agreed.

Reply:
Good point, shifted this and tweaked in 6be54268f8404a9c1b610f73d198532fe6824f3b